ansible/roles/passenger/tasks/main.yml

```diff
@@ -48,7 +48,7 @@
       dest: /etc/apache2/ssl/intermediate.crt
     sudo: true
 
-  - name: Create Rails Apache config (SSL Enabled)
```

Comment:
Right now intoxiclock update server is running node with apache. What does this passenger role even do? Why isn't this just under the apache role?